### ISSUE TITLE
READ FIRST : Images not found, or update error message

### ISSUE THREAD

**Issue by alexbelgium** [alexbelgium/hassio-addons#177]
I deeply apologize. I enabled autobuild as recommended by the HA guidelines, and it brought 2 big issues :
- Multi-arch base images are not supported (it needs a pull --platform tag that is not implemented due to various potential issues, and the system is very complex to fork and customize)
- Upgrade from a locally built addon to an autobuilt addon is not supported : the addon still tries to locally build and then provides a failure message as it doesn't find a local image (bug reported) 

I have decided to revert the changes. This means that everyone that has installed the new versions pushed since 2 days will have to restore a backup, or uninstall the app and reinstall it...

Here is the full procedure : 
1. Manually refresh the repository by going in the repository section (the same place your added my custom repo to your system) and click on reload. This will refresh the repositories and allow you to install the new corrected addon versions (or wait 1 day)
2. if you have a backup : 
       - restore the backup
       - hopefully everything should work directly, as for most of my addons the data is stored out of the addon
       - DO NOT update BEFORE you had refreshed the repo, or you will face the issue again
3. if you didn't have a backup : 
      - uninstalling the app is the only solution
      - if you're lucky, everything could work directly, as for most of my addons the data is stored out of the addon
      - DO NOT update BEFORE you had refreshed the repo, or you will face the issue again

Thanks for your understanding.

**Comment by almico** [comment]
I landed here, planning to open an issue exactly for this reason 😆

**Comment by alexbelgium** [comment]
yep - sorry about that. I'm really disappointed that it didn't work out but there just seems to be too many issues for me to continue with the deployment... And reverting it will impact less people that moving forward and making everyone face issues each time an addon update !

**Comment by alexbelgium** [comment]
I added a bit more info in the initial post on how to go back to a workable system

**Comment by almico** [comment]
> there just seems to be too many issues for me to continue with the deployment

Does this mean that you are not going to update any addon any more?

**Comment by alexbelgium** [comment]
Ahah, I'll change my wording. What I meant is that there were too much issues to continue with autobuild, so I reverted to local builds :)

In parallel I've opened an issue on the supervisor github, and hopefully in the future I'll be again able to enable autobuild!

This repo is here to stay :)

**Comment by alexbelgium** [comment]
On, my autobuild now works for all.addons but we still need resolution on the updater bug

**Comment by almico** [comment]
I've just been given the option to upgrade (?) qBitTorrent and this is the log:

![image](https://user-images.githubusercontent.com/15247405/151309081-2907b16e-2a94-47ba-862c-7843a98502ae.png)

I have version 14.3.9.99202110311443-7435-01519b5e7ubuntu20.04.1-ls166-7, but I cannot see it in the log...

**Comment by alexbelgium** [comment]
Hi, actually my version nomenclature is based on upstream images. For example, the long name from 14.3.9 to ls166 corresponds to the name of the qbittorrent docker image that linuxserver has made available here : https://hub.docker.com/r/linuxserver/qBittorrent

When I'm developing new features or correcting bugs, I add it on top of the changelog (the dots without version above) and I increment the number after -

Bottom line, the new version should add the three elements in the dots, or correct a bug linked to their implementation. It's not perfect, but that's the best I can do with the time I can allocate to the project ;)

Good idea though I'll try to see where I could put this info for people to have it available.

**Comment by almico** [comment]
Wow! Fast response, indeed 😃
My main concern is that, in the log, there is no reference to the version I have currently installed. I'm saying this, because I do think my current version was created well after January, 7th.

**Comment by alexbelgium** [comment]
Totally, I updated yesterday the bullet one of the changelog I think to correct a bug that someone had with accessing webui

**Comment by almico** [comment]
Just to be sure I understood: the log in the screenshot above is as you expected it to be, and it is normal that it doesn't contain all previous releases of the add-on.
Is this correct?

**Comment by alexbelgium** [comment]
Yes, in a way we could say it shows main versions and that intermediate versions not shown are bugs fixes that can be skipped if you don't have issues

**Comment by almico** [comment]
Ah! Now I see 😃
I would consider, though, including only the latest bug fix release (with its corresponding date), instead of the current approach that lists mostly buggy releases.

**Comment by alexbelgium** [comment]
This is an automatic log, because I write all those addons for free in parallel of a full time job in an unrelated sector and a family. All previous versions of qbittorrent were robust, the latest bugs are only due to people changing by themselves options or being on semi supported systems like HA supervised. I can help with addon and support, but I'm just too busy to spend tons of time on changelog... Yesterday I released around 7 addon versions on 3 addons, to help with some bugs. So sorry, but the changelog will stay as is ;) thanks for the suggestion though

**Comment by almico** [comment]
Sorry if it appeared I was not appreciating your work. I really love what you do 💌 😃

**Comment by alexbelgium** [comment]
Link with HA supervisor issue logged (issue with switching from local to remote built addon) https://github.com/home-assistant/supervisor/issues/3414
